src/MovieManager.java

```diff
@@ -91,8 +91,9 @@ public void addMovie() {
     public void editMovie() {
         System.out.println("Enter movieID: ");
         String movieID = sc.next();
-        for (Movie movie : movies) {
+        for (Movie movie : movies)
```

Comment:
Where did the curly bracket go? Is there a syntax error here? 
```suggestion
        for (Movie movie : movies) {
```